src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
Would suggest using the "for..in" loop instead of this one..Thus, instead of "actions[i]" you could use more readable variable. The hint is below:
Array => word + "s";
Object => word

Reply:
Agree ))